tests/PINCache.xcodeproj/xcshareddata/xcschemes/iOS.xcscheme

```diff
@@ -29,6 +29,15 @@
       buildConfiguration = "Debug">
       <Testables>
       </Testables>
+      <MacroExpansion>
+         <BuildableReference
+            BuildableIdentifier = "primary"
+            BlueprintIdentifier = "6629FFEC1A66B512009C10BD"
+            BuildableName = "PINCache.framework"
+            BlueprintName = "iOS"
+            ReferencedContainer = "container:PINCache.xcodeproj">
+         </BuildableReference>
+      </MacroExpansion>
```

Comment:
Strange it gets commited. 
I was trying to run tests from iOS target by cmd+u, however this one should be reverted..